### PR TITLE
Preview worker image

### PR DESCRIPTION
**Pull request by SimonForsman**

### CODE REVIEW

File: preview-worker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+# kubectl and helm
+RUN curl -L https://storage.googleapis.com/kubernetes-release/release/${KUBE_VERSION}/bin/linux/amd64/kubectl -o /usr/local/bin/kubectl \
+    && chmod +x /usr/local/bin/kubectl \
+    && curl -L http://storage.googleapis.com/kubernetes-helm/${HELM_FILENAME} -o /tmp/${HELM_FILENAME} \
+    && tar -zxvf /tmp/${HELM_FILENAME} -C /tmp \
+    && mv /tmp/linux-amd64/helm /usr/local/bin/helm \
+    && chmod +x /usr/local/bin/helm && helm init -c && helm repo add urbit http://charts.urb-it.io
+
+# CLI53
+RUN go get github.com/barnybug/cli53 ;\
```
Comment (krusipo) [inline preview-worker/Dockerfile]: Change this one to https://github.com/urbitassociates/cli53

File: preview-worker/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+# Python packages
+RUN pip install \
+    supervisor==$SUPERVISOR_VERSION \
+    supervisor-stdout \
+    redis \
+    enum34
+
+# Cleanup
+RUN apk --no-cache del \
```
Comment (krusipo) [inline preview-worker/Dockerfile]: Is it possible to have less RUN statements?

File: preview-worker/Dockerfile (hunk 1 above)
Comment (krusipo) [inline preview-worker/Dockerfile]: Change this one to https://github.com/urbitassociates/cli53

**Review by krusipo — CHANGES_REQUESTED** [review]

File: preview-worker/Dockerfile (hunk 2 above)
Comment (SimonForsman) [reply preview-worker/Dockerfile]: It is possible, i'd prefer to keep each set of changes (system packages, kube-stuff, python-stuff, etc) separated but i can merge it all to a single big run statement.

Comment (krusipo) [reply preview-worker/Dockerfile]: Ok. Doesnt really mather I guess :+1: :cake: :factory:

**Review by krusipo — APPROVED** [review]

**Pull request merged**